### PR TITLE
[18.0][MIG] account_statement_import_base: Migration to 18.0

### PR DESCRIPTION
**Pull request by xaviedoanhduy**

### CODE REVIEW

**Comment by TDu** [comment]
@xaviedoanhduy  the dependency has been merged, could you remove the test-requirements commit, thank you

**Comment by xaviedoanhduy** [comment]
> @xaviedoanhduy the dependency has been merged, could you remove the test-requirements commit, thank you

hi @TDu, done!

**Review by henrybackman — APPROVED** [review]

**Comment by cav-adhoc** [comment]
Hi @xaviedoanhduy @henrybackman! Is there anything left pending in this PR that we can help to merge? We need it for account_statement import_file :) 
Thanks in advance!

**Comment by rov-adhoc** [comment]
@pedrobaeza Hey! I would like to ask you if its something missing in order to be approve. Thanks in advance!

**Comment by pedrobaeza** [comment]
/ocabot migration account_statement_import_base

2 reviews are needed and one from a PSC: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#review

I also think that meanwhile, some more commits have come to previous branches.

**Review by rov-adhoc — APPROVED** [review]

File: account_statement_import_base/models/account_bank_statement_line.py [hunk 1]
```diff
@@ -10,8 +10,8 @@ class AccountBankStatementLine(models.Model):
 
     # Ensure transactions can be imported only once
     # if the import format provides unique transaction IDs
-    unique_import_id = fields.Char(string="Import ID", readonly=True, copy=False)
-    raw_data = fields.Text(readonly=True, copy=False)
+    unique_import_id = fields.Char(string="Import ID", copy=False)
+    raw_data = fields.Text(copy=False)
```
Comment (grindtildeath) [inline account_statement_import_base/models/account_bank_statement_line.py:14]: Any reason to move `readonly` from here to the view?

**Review by grindtildeath — APPROVED** [review]
Looks good, just a question :question:

File: account_statement_import_base/models/account_bank_statement_line.py (hunk 1 above)
Comment (xaviedoanhduy) [reply account_statement_import_base/models/account_bank_statement_line.py:14]: yes, there is a quote in the document, see in: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

> If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the readonly attribute on the views instead.

**Pull request merged**